.evergreen/config.yml

```diff
@@ -222,7 +222,7 @@ functions:
           ATLAS_ADMIN_API_USERNAME: ${atlas_admin_api_username}
           ATLAS_ADMIN_API_PASSWORD: ${atlas_admin_api_password}
           CLUSTER_NAME_SALT: ${build_id}
-        include_expansions_in_env: [DRIVER_DIRNAME]
+        include_expansions_in_env: [DRIVER_DIRNAME, PYTHON_BIN_DIR, ASYNC, FRAMEWORK, JAVA_HOME]
```

Comment:
I think we can go back to `add_expansions_to_env: true` for the commands that use the atlas creds.

Reply:
I quite like the explicitness of `include_expansions_in_env` it ensures we know _exactly_ what's required of a script.